### PR TITLE
Allow code and strikethrough formatting

### PR DESCRIPTION
**Pull request by oliverlloyd**
## What does this change?
Adds the ability to format text as `code` or ~~strikethough~~

## Why?
I read this article using my `iPhone 5c` and ~~quite liked~~, ~~loved~~,  adored it

## Depends on
https://github.com/guardian/discussion-api/pull/652

## Link to supporting Trello card
https://trello.com/c/h9gwb9qk/1430-strikethrough-formatting
&
https://trello.com/c/zy8l7sJh/1431-code-formatting

### CODE REVIEW

File: src/components/CommentForm/CommentForm.tsx [hunk 1]
```diff
@@ -475,6 +477,24 @@ export const CommentForm = ({
                             >
                                 i
                             </button>
+                            <button
```
Comment (gtrufitt) [inline src/components/CommentForm/CommentForm.tsx:480]: Could you add the Ophan attributes for this? Think the examples are in master as can't see them in the buttons above

Comment (oliverlloyd) [reply src/components/CommentForm/CommentForm.tsx:480]: If I look on frontend these button are li tags with no ophan stuff on them

```
<li class="d-comment-box__formatting-bold" title="Bold">B</li>
```

Clicking does send an ophan event but it seems to be a generic one, with no extra params, that gets sent on every click event, anywhere within discussion

```
https://ophan.theguardian.com/img/2?viewId=k9887d5ufcybupkecl0d&clickComponent=discussion
```

Comment (gtrufitt) [reply src/components/CommentForm/CommentForm.tsx:480]: Yup, it's new tracking, see https://github.com/guardian/discussion-rendering/pull/221/files#diff-224102f30975c2ad7f39e9be6c1c8d76R472

Comment (oliverlloyd) [reply src/components/CommentForm/CommentForm.tsx:480]: Oh! discussion `master`! Gotcha

Comment (oliverlloyd) [reply src/components/CommentForm/CommentForm.tsx:488]: Done.

Nice, now I get to see how popular my new formatting options are!

**Comment by oliverlloyd** [comment]
> would be cool to add a story with these style element in a message

Nice idea. It is so.

**Comment by oliverlloyd** [comment]
> Awesome! would could be cool to add `bold`, `italic`, `link` and `quote`

👍

**Pull request merged**